package.json

```diff
@@ -28,13 +28,13 @@
     "watch:jade": "chokidar 'src/index.jade' -c 'npm run build:jade'",
     "watch:scss": "chokidar 'src/scss/**/*.scss' -c 'npm run build:scss'",
     "test": "npm run test:sass",
-    "test:sass": "sass-lint -c node_modules/mofo-style/.sass-lint.yml --verbose --no-exit src/scss/**/*.scss",
+    "test:sass": "sass-lint -c .sass-lint.yml --verbose --no-exit src/scss/**/*.scss",
```

Comment:
Oh, .sass-lint was in the gitignore. That's weird. Fixing momentarily.

I added it for 2 reasons- 
* I didn't want to track the `mixins` file since it's a copy/paste of bootstrap code that I want to be able to easily diff in the future
* I wanted to change the "all selectors on different lines" rule, as it made `type.scss` really messy.